README.md

```diff
@@ -311,6 +311,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Lint pull request name with commitlint (Awesome if you squash merge !)](https://github.com/JulienKode/pull-request-name-linter-action)
 - [Block PR merges when Checks for target branches are failing](https://github.com/cirrus-actions/branch-guard)
 - [Get generated static site screeshots updated by Pull Request](https://github.com/ssowonny/diff-pages-action)
+- [Add labels depending if the Pull Request still in progress](https://github.com/AlbertHernandez/working-label-action)
```

Comment:
Please use Title Casing, there is a generator on this page: https://github.com/sdras/awesome-actions/blob/master/contributing.md

Reply:
@jessicalostinspace Done 6d5f5e06bc87f92b1cb47bed2074476f28b5fd16